src/fdb5/api/FDBFactory.h

```diff
@@ -125,6 +128,8 @@ class FDBBase : private eckit::NonCopyable {
     ControlIdentifiers controlIdentifiers_;
 
     bool disabled_;
+
+    ArchiveCallback callback_;
```

Comment:
Where is the default value for this being set? And should that be nullptr, or should it be a NOP function?

Reply:
Added a NOP function